README.md

```diff
@@ -60,6 +60,22 @@ On the machine being removed, run
 etcdadm reset
 ```
 
+### Renewal certificates
```

Comment:
This took me a second to understand. We aren't renewing the CA, we're renewing the certs generated with the CA. 

this should be a little more friendly, how about something like this:


> Certificates are set to expire after <1 year?> by default. 
> To renew certificates, first ensure every node in the cluster has the CA root certificate. Then, on each node in, run `etcdadm certs renew`.

Reply:
+1 to @chuckha's suggestion.

I think it might help to explicitly mention that there are two certificates: peer and server.

Reply:
I will update the document.

Reply:
Thank you @pytimer!